Add files via upload

CatrinaAndrei opened the pull request.

CatrinaAndrei committed:
Add files via upload

adinabadila commented on main.html:
```diff
@@ -0,0 +1,127 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="style.css">
+     <link href="fonts.css" rel='stylesheet' type='text/css'>
```
Indentare gresita

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+.body{
+       margin:0px;
+       /* background-color:black */
```
nu e ok sa lasi comentarii cand dai push

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+.body{
+       margin:0px;
```
indentare gresita

adinabadila replied:
E cu un tab in plus si trebuie sa lasi spatiu dupa ":"

adinabadila commented on main.html:
```diff
@@ -0,0 +1,127 @@
+<body class="body">
+    <header class="header">
+        <div class="header__container">
+            <div class="logo__container">
```
aici ti-as sugera sa folosesti header__logo

adinabadila commented on main.html:
```diff
@@ -0,0 +1,127 @@
+                    <img class="logo__image" src="Logo.png" alt="Save the planet Logo">
+                </a>
+            </div>
+            <nav class="navbar">
```
daca ai mers mai departe cu nav__item si asa mai departe, parintele ar trebui sa aiba tot clasa de "nav"

adinabadila commented on main.html:
```diff
@@ -0,0 +1,127 @@
+                </a>
+            </div>
+            <nav class="navbar">
+                    <ul class="nav__items">
```
Nu e ok sa folosesti numele de clase la plural

adinabadila commented on main.html:
```diff
@@ -0,0 +1,127 @@
+            </div>
+            <nav class="navbar">
+                    <ul class="nav__items">
+                            <li class="nav__item--home">
```
Indentare gresita, la fel si la ul mai sus

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+    justify-content: space-between;
+}
+
+.logo__image{
```
Daca pui imagine, ca sa o faci responsive trebuie sa adaugi width: 100%, height: auto, mai sunt si alte trick-uri, poti intreba daca nu e ceva clar

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+    display:flex;
+    max-width: 504px;
+    max-height: 21px;
+    list-style:none;
```
nu uita de spatiu dupa ":" 😁

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+}
+
+.nav__items:hover::after{
+    content: " ";
```
indentare gresita

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+    list-style:none;
+}
+
+.nav__items:hover::after{
```
Vizual: bulinuta nu este unde trebuie, si e pus hover pe items, ar trebui defapt sa fie pe fiecare item in parte

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+   /* margin:10px; */
+}
+
+.nav__link,.nav__link--home,.nav__link--dropdown{
```
spatiu dupa virgula!

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+    font-family: "CircularStd-Book";
+    font-size: 18px;
+    letter-spacing: 0.92px;
+    line-height: 23px;
```
line-height ul ar trebui sa fie calculat
font-size (px) / line-height (px)
exemplu: poate sa iti dea 1.3 sau ceva de genul

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+    max-width:161px;
+    max-height:186px;
+    margin: auto;
+    
```
scoate randurile astea goale

adinabadila commented on main.html:
```diff
@@ -0,0 +1,127 @@
+                        </ul>
+                </div>
+            </nav>
+                <div class="dropdown__container">
```
si aici e indentarea gresita + parintele trebuie sa fie dropdown, apoi dropdown__container si mergi mai departe cu alte elemente pe care le mai ai

adinabadila commented on main.html:
```diff
@@ -0,0 +1,127 @@
+                </div>
+            </nav>
+                <div class="dropdown__container">
+                    <ul class="dropdown__items">
```
la fel, nu e ok sa folosesti plural la numele clasei, gandeste-te la altceva

adinabadila commented on main.html:
```diff
@@ -0,0 +1,127 @@
+                        </li>
+
+                        <li class="dropdown__item">
+                                <a class="dropdown__link" href="#">Project</a>
```
indentare gresita, fii mai atent la asta, e foarte important

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+    z-index: -1;
+    top: -190px;
+    position: relative;
+    height: 1080px;
```
ti-as sugera sa ajungi la height-ul potrivit al hero-ului prin padding, nu prin height fix

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+        max-width: 335px;
+        max-height: 20px;
+        margin-bottom: 0px;
+    }
```
arata foarte urat aici, trebuie sa lasi spatiu dupa fiecare clasa

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+        max-width: 584px;
+        max-height: 178px;
+        margin: 0px;
+        line-height: 178px;
```
ti-am scris mai sus cum sa calculezi line-height ul

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+.body{
```
oriunde dai inspect in pagina vine body-ul peste si nu poti selecta elementele, adauga un z-index foarte mic si un position relative, ca sa poti da inspect pe elemente

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+        margin-bottom:70px;
+    }
+
+    .hero__buttons {
```
nu folosi plural aici

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+        color: white;
+        font-size: 12px;
+        letter-spacing: 0.7px;
+        max-width: 447px;
```
si aici iti sugerez sa faci cu padding, nu cu max-width, da-i un padding-right pana arata cum vrei tu sa arate, si apoi ajustezi padding-ul pe responsive

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+
+    .hero__button--transparent:hover{
+        background:white;
+        color:#272626;
```
spatiu dupa :

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+   transform: rotate(-90deg);
+   background-color: white;
+   position: relative;
+   /* width: 100vw; */
```
nu ar trebui sa apara comentarii in PR, scoate-le inainte sa dai push

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+    
+/* padding-left: 55px; */
+    
+margin: 36px 197px 31.5px 197px;
```
nu folosi virgula la valorile astea

adinabadila commented on main.html:
```diff
@@ -0,0 +1,127 @@
+    <div class="about__container">
+        <div class="about__mission">
+            <h1 class="about__title">Our Mission</h1>
+            <div class="about__content">Our wildlife is important to us
```
nu e corect folosit tag-ul de div aici, astea sunt clar niste paragrafe

adinabadila commented on main.html:
```diff
@@ -0,0 +1,127 @@
+        <div class="about__gallery">
+            <div class="about__photos">
+                <table class="tg">
+                    
```
scoate randul liber de aici

adinabadila commented on main.html:
```diff
@@ -0,0 +1,127 @@
+        </div>
+        <div class="about__gallery">
+            <div class="about__photos">
+                <table class="tg">
```
aici iti sugerez sa nu folosesti tabele.. joaca-te putin cu flex-ul aici, si hint: o sa ai nevoie de mai multe containere aici

adinabadila commented on main.html:
```diff
@@ -0,0 +1,127 @@
+                            </div>      
+                         </div>
+                        </td>
+                        <td class="photo__sheep td" colspan="3"><img class="photo__sheep" alt="herd of sheeps" src="about-sheep.jpg"></td>
```
niciodata nu le pui pe acelasi rand

adinabadila commented on main.html:
```diff
@@ -0,0 +1,127 @@
+                    </tr>
+                    <tr>
+                        <td class="photo__lake td" colspan="2"><img class="photo__lake" alt="upper view of lake" src="about-lake.jpg"></td>
+                        <td class="photo__cliff td"><img class="photo__cliff" alt="upper view o a cliff with a castle on it" src="about-cliff.jpg"></td>
```
nu ar trebui sa ai aceasi clasa si pe td si pe img

CatrinaAndrei replied:
Nu e aceeasi clasa,cea din tabel are "td" in plus,dar o sa modific.

adinabadila commented on main.html:
```diff
@@ -0,0 +1,127 @@
+                        <td class="photo__lake td" colspan="2"><img class="photo__lake" alt="upper view of lake" src="about-lake.jpg"></td>
+                        <td class="photo__cliff td"><img class="photo__cliff" alt="upper view o a cliff with a castle on it" src="about-cliff.jpg"></td>
+                      </tr>                     
+                
```
rand liber again🙃

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+ }
+
+ .card__title::before{
+     content:" ";
```
indentare gresita

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+
+ }
+
+.hero__button--card {
```
indentare gresita si aici

adinabadila commented on style.css:
```diff
@@ -0,0 +1,468 @@
+    /* background: radial-gradient(64.5% 64.5% at 96.09% -12.84%, rgba(0, 0, 0, 0.0001) 0%, rgba(5, 4, 2, 0.6) 100%); */
+}
+
+.tg{
```
foarte nesugestiv numele clasei

adinabadila reviewed (changes requested):
Vizual: lucrurile inca nu sunt la locul lor pe partea de desktop.
La meniu:
- hover-ul de pe fiecare item nu exista, meniul de dropdown nu functioneaza
- padding-urile dintre elemente nu sunt corecte
- pe hero ar trebui sa existe si partea pentru Slider, imaginea de hero se va schimba prin Slider, nu trebuie sa faci partea asta de JS acum, dar trebuie sa ai totul pregatit pentru asta

CatrinaAndrei committed:
Rezolvare request-uri

Meniul de dropdown din cate am inteles trebuie facut in javascript,asa ca doar l-am pus acolo deocamdata.
